examples/body-to-query-params-server/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "body-to-query-params-server",
+  "version": "1.0.0",
+  "description": "A simple server that recevies a JSON payload, and makes an outbound API request with some of the JSON object properties set to query parameters",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
we can delete this. Also, you may want to add a `start` script that just does `node app.js`

Reply:
Just trying to get this out for now

Reply:
Removed the test script